Windows/PerfOptions.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, layer_size=None, object_type='None', parent=None):
             self.checkBox.setChecked(True)
 
         self.decimate = 1
-        self.averaging = 1
+        self.subsampling = 1
```

Comment:
Ok so averaging => subsampling is just refactor -> name change?

Reply:
Yup, averaging is deprecated 📦